Source/WebInspectorUI/UserInterface/Views/DOMTreeContentView.css

```diff
@@ -85,3 +85,27 @@ body:not(.window-inactive, .window-docked-inactive) .content-view.dom-tree .tree
 .content-view.dom-tree .tree-outline.dom li.expanded .status-image.breakpoint.subtree {
     display: none;
 }
+
+.user-preferences-content {
+    display: grid;
+    padding: 5px;
+    gap: 4px;
+    grid-template-columns: auto auto;
+}
+
+.user-preferences-content > h5 {
```

Comment:
Why not just use `h1` here and style it appropriately, since this is the top-level type of header inside this popover?

Reply:
Done.